packages/api-generator/src/index.ts

```diff
@@ -63,6 +73,67 @@ const run = async () => {
     }).filter(Boolean)
   )
 
+  // Missing descriptions
+  if (argv.missingDescriptions) {
+    const translations: { [filename in string]?: TranslationData } = {}
+
+    async function readData (filename: string): Promise<TranslationData> {
+      if (!(filename in translations)) {
+        try {
+          const data = JSON.parse(await fs.readFile(filename, 'utf-8'))
+
+          for (const type of ['props', 'events', 'slots', 'exposed']) {
+            for (const item in data[type] ?? {}) {
+              if (data[type][item].startsWith('MISSING DESCRIPTION')) {
+                delete data[type][item]
+              }
+            }
+          }
+
+          translations[filename] = data
+        } catch (e) {
+          translations[filename] = {}
+        }
+      }
+
+      return translations[filename]
+    }
+
+    for (const index in componentData) {
```

Comment:
This is just personal preference so take it as a suggestion only (although I do wonder if we use in anywhere else), but I think I prefer a regular for loop or .forEach instead of for .. in for arrays. I think it's an unnecessary possible confusion of arrays and objects.